### DISCUSSION TITLE
我想把数据从 mongodb 迁移到 mysql , 要怎么迁移？Q:

### DISCUSSION THREAD

**Discussion by jerryc127** [walinejs/waline#1957] [Q&A]
### Describe the bug | 问题描述

我想把数据从 mongodb 迁移到 mysql , 要怎么迁移？

### Website URL | 问题网站

xx

### Where your waline deploy? | 服务部署在哪里？

Vercel (Default)

### Where your comment data store? | 数据存储在哪里？

MySQL

**Comment by lizheming** [comment]
Waline 后台导出数据再导入应该就行

**Reply by jerryc127** [reply]
不行 导入的数据 所有的时间都是显示导入那一刻的时间

**Reply by lizheming** [reply]
提个 Bug 吧
